packages/examples/src/registry.json

```diff
@@ -530,5 +530,9 @@
   "alloy-models/generic": {
     "name": "Alloy: Generic Style",
     "gallery": false
-  }
+  },
+  "Dynamics/Lyapunov": {
+    "name" : "Lyapunov Exponent",
```

Comment:
```suggestion
    "name": "Lyapunov Exponent",
```

Reply:
another small change to make the auto-formatter ([prettier](https://prettier.io/)) happy. We usually run `yarn format` or rely on the IDE to standardize code style. Also mentioned in the [docs](https://github.com/penrose/penrose/blob/main/CONTRIBUTING.md#opening-a-pull-request-pr)

Reply:
totally understood! thanks for going through the troubles. excited to have your diagram in the gallery!